File: foodfast-project/API/API.Data/Models/Review.cs
using System;
using System.Collections.Generic;

namespace API.Data.Models;

public partial class Review
{
    public long Id { get; set; }

    public string Author { get; set; } = null!;

    public long Score { get; set; }

    public string? Comment { get; set; }

    public long FkRestaurantidRestaurant { get; set; }
}

File: foodfast-project/src/models/Review.ts

export interface IReviewItem {
    id: string;
    author: string;
    score: number;
    comment: string;
}

export class Review implements IReviewItem {
    public id: string;
    public author: string;
    public score: number;
    public comment: string;

    public constructor(data: IReviewItem) {
        this.initFromData(data);
    }

    public initFromData(data: IReviewItem) {
        this.id = data.id;
        this.author = data.author;
        this.score = data.score;
        this.comment = data.comment;
    }
}